prac_05/hex_colours.py

```diff
@@ -0,0 +1,11 @@
+COLOUR_TO_CODE = {"Black": '#000000', "Brown": '#a52a2a', "Emerald": '#50c878',
+                  "FerrariRed": '#ff2800', "GoldenYellow": '#ffdf00', "GreenLizard": 'a7f432',
+                  "Iceberg": '71a6d2', "Lavender": 'e6e6fa', "Magenta": 'ff00ff', "OrangePeel": 'ff9f00'}
+
+colour_choice = input("Enter colour name: ").title().replace(' ', '')
```

Comment:
Smart use of .replace(). I'll have to do that myself. Otherwise, constants can be edited to match their environment.